src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/Create/ClassRoomCreate.tsx

```diff
@@ -17,31 +23,144 @@ const messages = defineMessages({
   },
   CreateClassroomLabel: {
     defaultMessage: 'Create Classroom',
-    description: 'Text heading create classroom.',
+    description: 'Button label to select classrooms',
     id: 'features.Contents.features.ClassRooms.Create.CreateClassroomLabel',
   },
+  SelectButtonLabel: {
+    defaultMessage: 'Select',
+    description: 'Text heading select classrooms.',
+    id: 'features.Contents.features.ClassRooms.Create.SelectButtonLabel',
+  },
+  DeleteButtonSingularLabel: {
+    defaultMessage: `Delete {value} classroom`,
+    description: 'Button label to delete single classroom.',
+    id: 'features.Contents.features.ClassRooms.Create.DeleteButtonSingularLabel',
+  },
+  DeleteButtonPluralLabel: {
+    defaultMessage: `Delete {value} classrooms`,
+    description: 'Button label to delete classrooms.',
+    id: 'features.Contents.features.ClassRooms.Create.DeleteButtonPluralLabel',
+  },
```

Comment:
This can be managed using only one message. Check the format documentation : https://formatjs.io/docs/core-concepts/icu-syntax